website/pages/docs/configuration/index.mdx

```diff
@@ -81,14 +81,22 @@ to specify where the configuration is.
   process's entire memory address space from spilling to disk and is therefore
   automatically disabled on unsupported platforms.
 
+  Disabling `mlock` is strongly recommended if using [integrated
+  storage](https://www.vaultproject.io/docs/internals/integrated-storage) due to
+  the fact that `mlock` does not interact well with memory mapped files such as
```

Comment:
Let's go into a little more detail here: when using `mlock` memory-mapped files are loaded into resident memory. This causes Vault to store the full dataset in memory and is particularly problematic when using larger-than-RAM datasets.

Reply:
Should be addressed by 8fa6fabb240ccf0ce4d740564ad63b4f5ad7972f